package.json

```diff
@@ -22,6 +24,7 @@
   "dependencies": {
     "async": "*",
     "cfn-response": "*",
+    "dotenv": "^4.0.0",
```

Comment:
Is it enough to have it in `devDependencies`?

Reply:
@kkuzmin You are correct.  I'll remove it.